3.11/release-notes-api-changes311.md

```diff
@@ -13,6 +13,20 @@ integrations for ArangoDB 3.11.
 
 ### Behavior changes
 
+### Graph API (Gharial)
+
+The following changes affect the behavior of the RESTful graph APIs at
+endpoints starting with path `/_api/gharial/`:
```

Comment:
The change only affects graph _creation_, doesn't it? Not all graph APIs at endpoints starting with `/_api/gharial`.